app/models/state_file_az_intake.rb

```diff
@@ -52,6 +52,8 @@
 #  primary_was_incarcerated                    :integer          default("unfilled"), not null
 #  prior_last_names                            :string
 #  raw_direct_file_data                        :text
+#  raw_direct_file_intake_data                 :jsonb
+#  raw_direct_file_tax_return_data             :text
```

Comment:
we probably don't want this since we decided against renaming the column

Reply:
good catch - removed it everywhere